src/views/v2/month/mobile-event/featured-image.php

```diff
@@ -0,0 +1,20 @@
+<?php
+/**
+ * View: Month View - Mobile Event Featured Image
+ *
+ * Override this template in your own theme by creating a file at:
+ * [your-theme]/tribe/events/views/v2/month/mobile-event/featured-image.php
+ *
+ * See more documentation about our views templating system.
+ *
+ * @link {INSERT_ARTCILE_LINK_HERE}
+ *
+ * @version TBD
+ *
+ */
+?>
+<div class="tribe-events-calendar-month__mobile-event-featured-image-wrapper">
+	<div class="tribe-events-calendar-month__mobile-event-featured-image tribe-common-c-image tribe-common-c-image--bg">
+		<div class="tribe-common-c-image__bg"></div>
```

Comment:
For any bgd image implementations, assuming they aren't decoration, but actual images for the event, we should be sure to add a `role="img" aria-label="alt text here"` on your bg image container.

Reply:
gotcha, will do!